doc/EMAIL.md

```diff
@@ -8,4 +8,38 @@ After configurations are done we are good to start the `mailman` server. For sta
 
 > ruby script/mailman_server
 
-Above command will start the server and then the mailman server will do polling in every 60 seconds(which is by default) to check for the incoming mail. However polling time can be changed by setting value to `Mailman.config.poll_interval`(more details can be found at its official [mailman user guide](https://github.com/mailman/mailman/blob/master/USER_GUIDE.md)). Logs for mailman can be seen oin Mailman log file which is present in `log` folder.
\ No newline at end of file
+Above command will start the server and then the mailman server will do polling in every 60 seconds(which is by default) to check for the incoming mail. However polling time can be changed by setting value to `Mailman.config.poll_interval`(more details can be found at its official [mailman user guide](https://github.com/mailman/mailman/blob/master/USER_GUIDE.md)). Logs for mailman can be seen oin Mailman log file which is present in `log` folder.
+    3.
+
+ 
+## Email Settings
+
+Users can modify **Email Settings** by visiting [https://publiclab.org/settings](https://publiclab.org/settings). Settings for each user are saved by using UserTags on their profiles. 
+For each setting, there is an associated tag with it. Below is the table showing associated tags with setting. 
+
+We consider absence of tag as true and presence of tag as false for setting, so whenever a user turn-off a setting, corressponding user-tag is generated and turning setting on again will delete the tag.  
+So, while notifying a user UserTag is checked in different files and user is notified depending on the presence or absence of the tag.
+
+| Email Settings  | User Tag |
+| ------------- | ------------- |
+| Notification by email for comments on your posts  | notify-comment-direct:false |
```

Comment:
Looks great, maybe we can have a column like `Default` showing the default setting, and the `User Tag` could say `User tag to override default` ? What do you think?

Reply:
and, i like the idea of linking to the lines of code!

Reply:
ok, I will make the changes. Thanks!